src/commands/get_prefix.rs

```diff
@@ -0,0 +1,28 @@
+use serenity::framework::standard::macros::command;
+use serenity::framework::standard::{Args, CommandResult};
+use serenity::{model::prelude::Message, prelude::Context};
+
+use crate::commands::send_message;
+use crate::db::guilds::GuildRepo;
+use crate::GlobalState;
+
+#[command]
+#[required_permissions(ADMINISTRATOR)]
+async fn get_prefix(ctx: &Context, msg: &Message, _args: Args) -> CommandResult {
+    let data_read = ctx.data.read().await;
+    if let Some(global_state) = data_read.get::<GlobalState>() {
+        let global_state = global_state.guilds.lock().await;
+        let prefix = global_state
+            .get_prefix(
+                if let Some(guild_id) = msg.guild_id {
+                    guild_id.0 as i64
+                } else {
+                    0
```

Comment:
if the guild id is inside the msg it's okay, but if we don't have it, we should return an error, look how I do in set_text_multiplier to return an error.
Using the 0 value would looks a 0 id-ed guild which will return in an error